packages/@cdktf/hcl2cdk/test/expressions.test.ts

```diff
@@ -23,13 +23,13 @@ describe("expressions", () => {
     it("finds no references in literals", () => {
       expect(
         extractReferencesFromExpression("nothingtobeseen", nodeIds)
-      ).toEqual([]);
+      ).resolves.toEqual([]);
```

Comment:
Does jest track these kinds of checks? Or would this need an async test and an `await expect(...`?

Reply:
Yes it does, you can either await directly or have an expect on a promise with the resolves property, both work. but with await you have to return the expect (I think)